src/type.zig

```diff
@@ -3421,7 +3421,7 @@ pub const Type = extern union {
         anyerror_void_error_union,
         generic_poison,
         /// This is a special type for variadic parameters of a function call.
-        /// Casts to it will validate that the type can be passed to a c calling convetion function.
+        /// Casts to it will validate that the type can be passed to a c calling convection function.
```

Comment:
"Convention", not "convection".

Reply:
Oops, when Google Sheets made that suggestion, I was pretty sure it was wrong, I should have looked. Thanks, fixed.